app/src/components/market/sections/market_create/steps/items/create_scalar_market.tsx

```diff
@@ -125,6 +120,30 @@ export const CreateScalarMarket = (props: Props) => {
     upperBound,
   } = props
 
+  const [lowerBoundError, setLowerBoundError] = useState('')
+  const [startingPointError, setStartingPointError] = useState('')
+  const [upperBoundError, setUpperBoundError] = useState('')
+
+  useEffect(() => {
+    Number(lowerBound) < 0
+      ? setLowerBoundError('Value cannot be negative')
+      : upperBound && !upperBound.eq(0) && lowerBound?.gt(upperBound)
```

Comment:
Should also make sure we're using `Zero` here instead of `0` as with other places

Reply:
My bad btw, I think I suggested using `0` oops